XXXXX-template.md

```diff
@@ -1,85 +1,262 @@
+
+- Does this change economic incentives or token economics?
+- What are the cost implications for different network participants?
+- How might this affect network value accrual?
+
+## Compatibility
+
+### Backward Compatibility
+
+- Is this change backward compatible?
+- If not, what is the migration strategy?
+- How will legacy systems be supported?
+
+### Forward Compatibility
+
+- How does this design accommodate future enhancements?
+- What extensibility mechanisms are provided?
+
+### Ecosystem Integration
+
+- How will this work with existing tools and infrastructure?
+- What changes are needed in wallets, explorers, SDKs?
+- How does this interact with other protocols in the ecosystem?
+
+## Security Considerations
+
+- What are the security implications of this change?
+- What new attack vectors might be introduced?
+- How are these risks mitigated?
+- What security assumptions does this make?
+- Are there any cryptographic considerations?
+
+## Performance Implications (optional)
+
+### Computational Impact
+- Expected changes in CPU usage, memory consumption
+- Gas cost implications for smart contracts
+- Benchmarking methodology
+
+### Network Impact
+- Bandwidth requirements
+- Latency implications
+- Scalability considerations
+
+### Storage Impact
+- State growth implications
+- Archive node requirements
+
+## Implementation Plan
+
+### Development Phases
+
+Break down implementation into logical phases:
+1. Phase 1: Core implementation
+2. Phase 2: Integration and testing
+3. Phase 3: Deployment and migration
+
+### Testing Strategy
+
+- Unit testing approach
+- Integration testing requirements
+- Network testing on testnets
+- Security auditing requirements
+
+### Deployment Strategy
+
+- Rollout phases (testnet → mainnet)
+- Feature flags or gradual activation
+- Monitoring and rollback procedures
+
+### Maintenance
+
+- Who will maintain this code?
+- What ongoing operational requirements exist?
+- How will this be monitored and debugged?
+
+## Documentation and Examples
+
+### Tutorial and Usage Examples
+
+Provide end-to-end examples showing:
+- How developers will use new features
+- Integration patterns
+- Common use cases
+
+The examples should be written as user-facing documentation, not internal technical specs.
+
+### Migration Guide (optional - required only for breaking changes)
+
+If this changes existing functionality, provide:
+- Step-by-step migration instructions
+- Code examples showing before/after
+- Timeline for deprecation of old methods
+
+## Community Consensus
+
+### Discussion Summary
+
+Document community feedback and discussions:
+- Links to [forum discussions](https://forum.story.foundation/), Discord conversations
+- Summary of key feedback themes
+- How feedback was incorporated into the design
+
+### Stakeholder Impact
+
+- Which stakeholders are most affected?
+- What is their level of support or concern?
+- How were conflicts resolved?
+
+### Governance Considerations
```

Comment:
Governance process should be independent of individual SIPs, unless this is an SIP targeting governance.
Ideally there is a common governance process to settle which SIPs are included in a fork. For app/ecosystem standards it may not need voting for example.